default/requirements.txt

```diff
@@ -1,4 +1,6 @@
 aiofiles==0.5.0
 fastapi==0.59.0
+google-cloud-datastore==1.13.1
```

Comment:
Take a look at python-ndb (https://github.com/googleapis/python-ndb). I prefer the interface ndb provides, defining the model as a class.